Rakefile

```diff
@@ -51,6 +51,11 @@ def install_pods_in_example_project()
   $?.exitstatus == 0
 end
 
+def check_pristine()
+  changes = `git status --porcelain | grep -Ev "CHANGELOG|UNRELEASED"`.lines
+  result = changes.length < 1
```

Comment:
This can just be `changes.length == 0`

Reply:
![Good point](https://media1.giphy.com/media/xzMs8O1KQh17i/200w.gif?cid=c94812d05bfd134b717a3069774da80b)